<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPushButton" name="Main_vk">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(177, 99, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>Главная</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="VK">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(177, 99, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>API функции VK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TG_vk">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(177, 99, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>API функции TG</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;VK &amp;amp; TG functions &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>1. Включение вечного онлайна</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>2. Добавление аккаунта в бан</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>3. Вывод списка друзей</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>4. Получение статуса пользователя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>5. Изменение статуса пользователя</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Выберете номер функции</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nums">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 99, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="function">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(177, 99, 255);
font: 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
